src/Microsoft.TemplateEngine.Orchestrator.RunnableProjects/ConfigModel/ParameterSymbol.cs

```diff
@@ -62,6 +62,11 @@ internal ParameterSymbol(string name, JObject jObject, string? defaultOverride)
                 DefaultIfOptionWithoutValue = "true";
             }
 
+            if (DefaultValue == null && IsRequired != true)
```

Comment:
is `IsRequired` needed here? 
does it change any behavior?

Reply:
`IsRequired` indicates that value needs to be specified. Dafult is not needed. So it can technically be removed here from the condition without any impact. I just wanted the code not to create impression that the `Default` is created *and applied* even for required parameters.